pages/projects_page.py

```diff
@@ -80,6 +82,34 @@ def rename_workspace(self, old_name, new_name):
                 self.click(self._CONFIRMATION_BUTTON)
                 break
 
+    @allure.step("Start a new project")
+    def create_new_project(self):
+        if self.is_elem_displayed(self._driver.find_element(*self._START_BUTTON)):
+            self.click(self._START_BUTTON)
+        elif self.is_elem_displayed(self._driver.find_element(*self._CREATE_PROJECT_BUTTON)):
+            self.click(self._CREATE_NEW_WORKSPACE_BUTTON)
+
+    @allure.step("Search for project {project_name}")
+    def search_project(self, project_name):
+        self.click(self._SEARCH_BUTTON)
+        self.fill_text(self._SEARCH_FIELD, project_name)
+
+    @allure.step("Delete or cancel deletion of project {project_name}")
+    def delete_project(self, project_name, status="confirm"):
+        projects = self._wait.until(expected_conditions.visibility_of_all_elements_located(self._PROJECTS_BLOCK))
+        deleted_project = None
+        for project in projects:
+            if project_name in project.text:
+                deleted_project = project
+                self.click_drop_down_menu(project)
+                project.find_element(By.XPATH, "//button[text()='Delete Project']").click()
+                break
+        if status == "cancel":
```

Comment:
create a class for status and pass as a fixture 
https://docs.python.org/3.4/library/enum.html

Reply:
This argument (status) is used only in this pom method. 
Do you think it is really needed to create an Enum class with only these two values? (confirm, cancel)

Reply:
yes for learning purpose

Reply:
Then this should be created in a different file as a helper module and be imported to POM.
POM methods cannot take fixtures as arguments because they are not registered as test methods.
I will do it today